Decent.Minecraft.Client/Blocks/Coal.cs

```diff
@@ -11,6 +14,9 @@ protected Coal(bool charred) : base(BlockType.Coal)
         protected bool IsCharred { get; }
     }
 
+    /// <summary>
+    /// <a href="http://minecraft.gamepedia.com/Coal_Ore">Gamepedia link</a>.
+    /// </summary>
     public class Charcoal : Coal
```

Comment:
Just noticed this one... There is no Charcoal block in the game, but the Coal ore block is missing.